spec/factories/call_off_contract.rb

```diff
@@ -6,6 +6,7 @@
     uplift_amount { 100 }
     recruitment_target { 2000 }
     set_up_fee { 149_651 }
+    lead_provider { build(:lead_provider, cpd_lead_provider: build(:cpd_lead_provider)) }
```

Comment:
I'm wondering if this new one from the build will get persisted or just end up in-memory